server/controllers/content.js

```diff
@@ -414,10 +416,36 @@ export function renderNewsletterPage(ctx, next) {
 export async function searchForDrupalRedirect(ctx, next) {
   const {path} = ctx.params;
   const page = await getPageFromDrupalPath(ctx.request, `/${path}`);
+  const baseGaPayload = {
+    v: 1,
+    tid: 'UA-55614-6',
+    cid: 555,
+    t: 'event',
+    ec: 'Server',
+    ea: 'Redirect'
+  };
+  const gaPath = 'https://www.google-analytics.com/collect';
 
   if (page) {
+    superagent.post(gaPath)
+      .type('form')
+      .send(Object.assign({}, baseGaPayload, {el: `from:${path} to: /pages/${page.id}`}));
+
     ctx.status = 301;
     ctx.redirect(`/pages/${page.id}`);
+  } else {
+    const exhibition = await getExhibitionFromDrupalPath(ctx.request, `/${path}`);
+
+    if (exhibition) {
+      superagent.post(gaPath)
+        .type('form')
+        .send(Object.assign({}, baseGaPayload, {el: `from:${path} to: /exhibitions/${exhibition.id}`}));
```

Comment:
Would it be easier to read in `ga` to just have the URLs we're redirecting?
Seems like that the info we would be using to make any decisions.

Reply:
Probably, yeah. Will update.